content/events/21-22/320_barbecue_studententuin.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Barbecue in studententuin!
+description: Zomervakantie! Barbecue, spelletjes en dan Gentse Feesten!
+created_at: xx-06-2022
+time: '16-07-2022 xx:xx'
+end: '16-07-2022 xx:xx'
+location: Voortuin van de studententuin (aan S5, Campus Sterre)
+locationlink: "Studententuin voortuin, 9000 Ghent"
+banner: "https://images0.persgroep.net/rcs/eBWqGxiUnWGoG-SXwY87Y5k2NYE/diocontent/177694046/_fill/1200/630/?appId=21791a8992982cd8da851550a453bd7f&quality=0.7"
+header_text_background: true
+tags:
+  - Barbecue
+  - Studententuin
+
+---
+
+Omdat we elkaar niet de hele zomervakantie kunnen missen, organiseren we een zomers evenement!
+Op 16 juli komen we samen voor een lekkere barbecue. Er zullen spelletjes zijn en niet toevallig is het dan Gentse Feesten!
+Na de barbecue kunnen we naar daar afzakken.
+
+<Vlees>
```

Comment:
Als we er een potluck van maken, dan is het vrij makkelijk, dan moeten we geen inschrijvingen etc doen